src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,55 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * This class is using a decorator pattern for representing
+         * a one-to many header-body relation.
+         */
+        private static class RqMultipartRequest implements Request {
+
+            /**
+             * Request object. Holds a value for the header.
+             */
+            private final Request req;
+
+            /**
+             * Holding multiple request body parts.
+             */
+            private final Request[] dispositions;
+
+            /**
+             * The Constructor for the class.
+             * @param req The Request object
+             * @param dispositions The disposition list
```

Comment:
@sebing minor: this is not a list, it's rather array. Maybe `The array of dispositions` or more generic `The sequence of dispositions` would be better here?